app/assets/stylesheets/settings.scss

```diff
@@ -545,30 +550,29 @@ $input-width: 650px;
         &:hover,
         &:focus {
           background: $green;
+          &.danger-button {
+            background: $red;
+          }
         }
         &:disabled {
           cursor: wait;
           background: white url(image-src('loading-ellipsis.svg')) no-repeat
             center center;
           background-size: 75px;
         }
+        a {
+          color: $black;
+        }
+      }
+      &.response-template-row-name {
+        button,
+        a[role='button'] {
+          width: 80px;
+          margin-left: 2.5px;
+          margin-right: 2.5px;
+          text-align: center;
+        }
       }
-    }
-    .github-repo-fork {
-      margin-left: 5px;
-      font-weight: 400;
-      background: $purple;
-      color: $bold-blue;
-      border-radius: 3px;
-      padding: 1px 6px;
-      font-size: 0.9em;
-    }
-    .github-repo-featured-indicator {
-      display: inline-block;
-      float: right;
-      background: $green;
-      color: white;
-      border: 0px;
```

Comment:
This is dead code.

Reply:
Are you sure? It seems to be used here:

https://github.com/thepracticaldev/dev.to/blob/81fff248dd953f873161ed276ddb7f0ab576bcf8/app/javascript/githubRepos/singleRepo.jsx#L13-L15

This may have introduced #8834.

Reply:
Ahh yeah, I think you're right. Good catch!